### PR TITLE
code_0x8017FEB0 OK

### PR DESCRIPTION
**Pull request by AngheloAlf**
~~I'm a bit unsure about `D_801E01E0`. It seems to be used only by `atan`. I don't know how to import that variable to code.~~

I also fixed the signature of `atan_flip`.

Update: 
- I renamed the functions and the file to match its OoT counterpart.
- I also added the renamed functions to `actorfixer.py`

### CODE REVIEW

**Comment by AngheloAlf** [comment]
This is fixed now!

File: src/code/code_0x8017FEB0.c [hunk 1]
```diff
@@ -1,12 +1,120 @@
...
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/code/code_0x8017FEB0/atans.asm")
+u16 atans_first_8th(f32 opposite, f32 adjacent) {
+    return atan_first_8th_array[(s32)((opposite / adjacent) * 0x400)];
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/code/code_0x8017FEB0/atan.asm")
+s16 atans(f32 opposite, f32 adjacent) {
+    s32 phi_v1;
```
Comment (hensldm) [inline src/code/code_0x8017FEB0.c:66]: `angle` seems like a better name then `phi_v1`.

**Review by hensldm — APPROVED** [review]
Looks good to me. Only saw one small thing.

**Review by Zelllll — CHANGES_REQUESTED** [review]
If these functions exist in oot they should probably be using the same names, and everything should be camelCase. Also the `atan_first_8th_array` should be `static u16 sAtanFirstEighth[]` or something instead

**Comment by AngheloAlf** [comment]
The equivalent file seems to be [sys_math_atan.c](https://github.com/zeldaret/oot/blob/master/src/code/sys_math_atan.c), so I'll rename the functions and the filename too

**Comment by AngheloAlf** [comment]
Update: 
- I renamed the functions and the file to match its OoT counterpart.
- I also added the renamed functions to `actorfixer.py`

**Review by Zelllll — APPROVED** [review]
Looks good now

**Review by Kenix3 — APPROVED** [review]
Looks great to me! Don't forget to run format.sh on this.

**Pull request merged**